src/controller/progress/tracking.service.ts

```diff
@@ -58,7 +58,7 @@ export class TrackingService {
       if (chapterExistsInModuleChapter.length != 0) {
         if (chapterExistsInChapterTracking.length == 0) {
           let chapterTracked;
```

Comment:
Hello @sukanyamohanty143 
Are there any changes in this file, or is it only a formatting?